ui_framework/doc_site/src/views/flex/flex_items.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import {
+  KuiFlexGroup,
+  KuiFlexItem,
+} from '../../../../components';
+
+export default () => (
+  <div>
```

Comment:
For examples with a single root element, like this one, we can get rid of the wrapper `<div>` to clean things up a bit.